package.json

```diff
@@ -26,7 +26,7 @@
     "react-dom": "18.2.0",
     "react-icons": "^4.4.0",
     "react-loader-spinner": "^5.2.0",
-    "starknet": "^4.4.2",
+    "starknet": "^4.17.1",
```

Comment:
why should we follow the dev branch of starknetjs?

Reply:
It's not the dev branch is it ?

Reply:
It's just the last version. 

Dev branch is @next